### DISCUSSION TITLE
I think it is very good!

### DISCUSSION THREAD

**Discussion by pgmrDohan** [community/community#3542] [Discussions] [answered]
Thank you!

**Comment by Kainichy** [comment, marked as answer]
Totally agree. I think GitHub Discussions can replace StackOverflow someday.

**Reply by pgmrDohan** [reply]
I think it's because we have a lot of users with StackOverflow, but github has more users than that.

**Comment by PranamBhat** [comment]
Hi @Developer-CoderK 

Kindly mark this as "**Answered**" and leave the discussion. Thank you.
